utils/luafunc_dup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/bash
+
+LST=`tempfile`
```

Comment:
```suggestion
LST=$(mktemp)
```
Apparently `tempfile` is deprecated and backtick notation is considered legacy.

Reply:
Damnit, I am feeling old, now.
But true, someone already told me that backtick notation was not used anymore, a long time ago.